libcloud/compute/drivers/outscale.py

```diff
@@ -0,0 +1,1109 @@
+        if keypair_name is not None:
+            data.update({"KeypairName": keypair_name})
+        if max_vms_count is not None:
+            data.update({"MaxVmsCount": max_vms_count})
+        if min_vms_count is not None:
+            data.update({"MinVmsCount": min_vms_count})
+        if nics is not None:
+            data.update({"Nics": nics})
+        if performance is not None:
+            data.update({"Performance": performance})
+        if placement is not None:
+            data.update({"Placement": placement})
+        if private_ips is not None:
+            data.update({"PrivateIps": private_ips})
+        if security_group_ids is not None:
+            data.update({"SecurityGroupIds": security_group_ids})
+        if security_groups is not None:
+            data.update({"SecurityGroups": security_groups})
+        if subnet_id is not None:
+            data.update({"SubnetId": subnet_id})
+        action = "CreateVms"
+        data = json.dumps(data)
+        signer = OSCRequestSignerAlgorithmV4(access_key=self.key,
+                                             access_secret=self.secret,
+                                             version=self.version,
+                                             connection=self.connection)
+        headers = signer.get_request_headers(action=action,
+                                             data=data,
+                                             service_name=self.service_name,
+                                             region=self.region)
+        endpoint = self._get_outscale_endpoint(self.region,
+                                               self.version,
+                                               action)
+        return requests.post(endpoint, data=data, headers=headers)
+
+    def reboot_node(self, node_ids):
+        """
+        Reboot instances.
+
+        :param      node_ids: the ID(s) of the VM(s)
+                    you want to reboot (required)
+        :type       node_ids: ``list``
+
+        :return: the rebooted instances
+        :rtype: ``dict``
+        """
+        action = "RebootVms"
+        data = json.dumps({"VmIds": node_ids})
+        signer = OSCRequestSignerAlgorithmV4(access_key=self.key,
+                                             access_secret=self.secret,
+                                             version=self.version,
+                                             connection=self.connection)
+        headers = signer.get_request_headers(action=action,
+                                             data=data,
+                                             service_name=self.service_name,
+                                             region=self.region)
+        endpoint = self._get_outscale_endpoint(self.region,
+                                               self.version,
+                                               action)
+        return requests.post(endpoint, data=data, headers=headers)
+
+    def list_nodes(self, data="{}"):
```

Comment:
To comply with the base Libcloud API, this method needs to return a list of ``Node`` objects.